HISTORY.rst

```diff
@@ -42,6 +42,8 @@ Unreleased Changes
       have been replaced with ``numpy.prod``.
       https://github.com/natcap/invest/issues/1410
     * Add support for python 3.11 (`#1103 <https://github.com/natcap/invest/issues/1103>`_)
+    * Vector geometry types will now be validated for all models
```

Comment:
Is this true? I see the update to `validation.py` but I don't see where each model would be taking advantage of the new param in `check_vector`.